products/mlengine/api.yaml

```diff
@@ -235,4 +235,338 @@ objects:
+        description: Required. The user-specified id of the job.
+        required: true
+      - !ruby/object:Api::Type::Time
+        name: 'createTime'
+        description: When the job was created.
+        output: true
+      - !ruby/object:Api::Type::Time
+        name: 'endTime'
+        description: When the job processing was completed.
+        output: true     
+      - !ruby/object:Api::Type::Enum
+        name: 'state'
+        description: The detailed state of a job.
+        values:
+          - STATE_UNSPECIFIED
+          - QUEUED
+          - PREPARING
+          - RUNNING
+          - SUCCEEDED
+          - FAILED
+          - CANCELLING
+          - CANCELLED
+        output: true     
+      - !ruby/object:Api::Type::String
+        name: 'errorMessage'
+        description: The details of a failure or a cancellation.
+        output: true                
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'labels'
+        description: |
+           One or more labels that you can add, to organize your jobs. Each label is a key-value pair, 
+           where both the key and the value are arbitrary Strings that you supply. 
+           An object containing a list of "key": value pairs. Example: { "name": "wrench", "mass": "1.3kg", "count": "3" }.
+      # no etag since it's input: true     
+      ## trainingInput starts here     
+      - !ruby/object:Api::Type::NestedObject
+        name: 'trainingInput'
+        description: Input parameters to create a training job.
+        output: false
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: 'scaleTier'
+            description: Specifies the machine types, the number of replicas for workers and parameter servers.
+            values:
+              - BASIC
+              - STANDARD_1
+              - PREMIUM_1
+              - BASIC_GPU
+              - BASIC_TPU
+              - CUSTOM
+            required: true
+          - !ruby/object:Api::Type::String
+            name: 'masterType'
+            description: Specifies the type of virtual machine to use for your training job's master worker  
+            required: false  
+          - !ruby/object:Api::Type::NestedObject
+            name: 'masterConfig'
+            description: The configuration for your master worker.
+            properties:
+              - !ruby/object:Api::Type::NestedObject
+                name: 'acceleratorConfig'
+                description: Represents the type and number of accelerators used by the replica
+                properties:
+                - !ruby/object:Api::Type::Integer
+                  name: 'count'
+                  description: The number of accelerators to attach to each machine running the job.
+                - !ruby/object:Api::Type::Enum
+                  name: 'type'
+                  description: The type of accelerator to use.
+                  values:
+                    - ACCELERATOR_TYPE_UNSPECIFIED
```

Comment:
I'm not super familiar with the product- is this value one that users might actually set?

Reply:
I have referred below link

https://cloud.google.com/ml-engine/reference/rest/v1/AcceleratorConfig
https://cloud.google.com/ml-engine/reference/rest/v1/AcceleratorType